apps/aechannel/src/aesc_channels.erl

```diff
@@ -57,20 +57,41 @@
 %%% Types
 %%%===================================================================
 
+-export_type([ channel/0
+             , id/0
+             , amount/0
+             , pubkey/0
+             , round/0
+             , solo_round/0
+             , is_active/0
+             , lock_period/0
+             , locked_until/0
+             , state_hash/0
+             , serialized/0
+             , seq_number/0
+             , payload/0
+             ]).
+
 -type id() :: aeser_id:id().
 -type pubkey() :: aec_keys:pubkey().
 -type amount() :: non_neg_integer().
 -type seq_number() :: non_neg_integer().
--type payload() :: aesc_offchain_tx:tx() | <<>>.
```

Comment:
This removal of `<<>>` is the only unclear thing to me.

Reply:
In this context the payload must be an off-chain transaction. Although in solo closing sequence we could use an empty payload, there are distinct functions in this module that handle this specific case. For example we have a function for solo close with [an empty payload](https://github.com/aeternity/aeternity/blob/943dcee32890c3fdd9074b94a66f7f8478414652/apps/aechannel/src/aesc_channels.erl#L123-L127) and yet another for [non-empty one](https://github.com/aeternity/aeternity/blob/943dcee32890c3fdd9074b94a66f7f8478414652/apps/aechannel/src/aesc_channels.erl#L129-L133). Note that the latter has a strong expectations for the `payload()` being passed to be explicitly an `aesc_offchain_tx:tx()`.